back/src/controllers/cra.controller.ts

```diff
@@ -0,0 +1,113 @@
+import { CraApplication } from "../domain/application/craApplication";
+import { CreateAbsenceDto } from "../Dto/CreateAbsenceDto";
+import { Absence } from "../domain/model/Absence";
+import { Body, Controller, Delete, Get, HttpException, HttpStatus, Param, Post } from "@nestjs/common";
+import { CreateActivityDto } from "../Dto/CreateActivityDto";
+import { Activity } from "../domain/model/Activity";
+import { deleteActivityAbsenceDto } from "@app/Dto/deleteActivityAbsenceDto";
+
+@Controller('cra')
+export class CraController {
+  constructor(private readonly craApp: CraApplication) { }
+  /*
+      @Post()
+      async createCra(@Body() createCraDto: CreateCraDto): Promise<CRA> {
```

Comment:
No dead code please. Also you have a duplicate @Post annotation due to the comment